Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /analyzer-lsp
 # TODO limit to prevent unnecessary rebuilds
 COPY  . /analyzer-lsp
 COPY --from=builder /analyzer-lsp/konveyor-analyzer /usr/bin/konveyor-analyzer
+COPY --from=builder /analyzer-lsp/external-providers/golang-external-provider/golang-external-provider /usr/bin/golang-external-provider
 COPY provider_container_settings.json /analyzer-lsp/provider_settings.json
 
-
-CMD ["/bin/bash", "-c", "go run main.go;cat output.yaml"]
+CMD ["/bin/bash", "-c", "golang-external-provider & disown && sleep 5; konveyor-analyzer --output-file output.yaml; cat output.yaml"]
```

Comment:
Do we still need to do this?

Reply:
I think I have the config set to address because we can not manage the ports yet probably need to add a follow-up?

Reply:
https://github.com/konveyor/analyzer-lsp/issues/166

Reply:
sounds good to me